tempesta_fw/ss_skb.c

```diff
@@ -602,9 +665,10 @@ static int
 __skb_fragment(struct sk_buff *skb, struct sk_buff *pskb,
 	       char *pspt, int len, TfwStr *it)
```

Comment:
```__skb_fragment()``` is always called with ```pskb == NULL```, the only one case where it's called with non-NULL ```pskb``` is recursive call. Isn't it better to introduce ```skb_fragment()``` which doesn't have ```pskb``` argument at all and just calls ```__skb_fragment()``` with NULL as ```pskb```?